scripts/konflux/component-squad.yaml

```diff
@@ -0,0 +1,185 @@
+# Component to Squad mapping configuration
+# Auto-generated from component-squad-owner.md
+
+squads:
+  server-foundation:
+    name: "Server Foundation"
+    components:
+      - addon-manager-mce
+      - cluster-proxy-addon-mce
+      - cluster-proxy-mce
+      - clusterlifecycle-state-metrics-mce
+      - managed-serviceaccount-mce
+      - managedcluster-import-controller-mce
+      - multicloud-manager-mce
+      - placement-mce
+      - registration-mce
+      - registration-operator-mce
+      - work-mce
+      - klusterlet-addon-controller-acm
+
+  install:
+    name: "Install"
+    components:
+      - backplane-must-gather-mce
+      - backplane-operator-mce
+      - discovery-operator-mce
+      - mce-operator-bundle-mce
+      - acm-operator-bundle-acm
+      - multiclusterhub-operator-acm
+
+  hypershift:
+    name: "Hypershift"
+    components:
+      - cluster-api-provider-agent-mce
+      - cluster-api-provider-kubevirt-mce
+      - hypershift-addon-operator-mce
+      - hypershift-cli-mce
+      - hypershift-release-mce
+
+  capi:
+    name: "CAPI"
+    components:
+      - cluster-api-provider-aws-mce
+      - cluster-api-webhook-config-mce
+      - ose-cluster-api-rhel9
+      - ose-aws-cluster-api-controllers-rhel9
+      - ose-baremetal-cluster-api-controllers-rhel9
+
+  assisted-installer:
+    name: "Assisted Installer"
+    components:
+      - cluster-api-provider-openshift-assisted-bootstrap-mce
+      - cluster-api-provider-openshift-assisted-control-plane-mce
+      - assisted-image-service
+      - assisted-installer
+      - assisted-installer-agent
+      - assisted-installer-controller
+      - assisted-service-8
+      - assisted-service-9
+      - postgresql-12
+
+  cluster-lifecycle:
+    name: "Cluster Lifecycle"
+    components:
+      - cluster-curator-controller-mce
+      - cluster-image-set-controller-mce
+      - clusterclaims-controller-mce
+      - provider-credential-controller-mce
+
+  console:
+    name: "Console"
+    components:
+      - console-mce-mce
+      - console-acm
+
+  hive:
+    name: "Hive"
+    components:
+      - hive-mce
+
+  ibio:
+    name: "IBIO"
+    components:
+      - image-based-install-operator-mce
+
+  hypershift-observability:
```

Comment:
This is converted from https://docs.google.com/spreadsheets/d/1nj0PnLlLcSJ-uwb469Mc3lqI98Hh7a7sjKR31IVao_I/edit?gid=912270968#gid=912270968 , 

I still do not know who is the owner of the component `kube-rbac-proxy`? hypershift or obs?